modules/blockbuilder/blockbuilder.go

```diff
@@ -387,11 +387,11 @@ outer:
 	level.Info(b.logger).Log(
 		"msg", "successfully committed offset to kafka",
 		"partition", ps.partition,
-		"last_record", lastRec.Offset,
+		"commit_offset", lastRec.Offset+1,
```

Comment:
Should this be ` lastRec.Offset` ?. This is the committed one. A bit philosophical, nobody is going to inspect this log statement :P

Reply:
Commit is actually `lastRec.Offset+1`, that's the tricky part. Commit is the first unread message, rather than the last read message.

In terms of logging, not sure what's more relevant to the operator tbh.

Reply:
Cool I like the offset.At, it abstracts all of this!